yjit/src/cruby.rs

```diff
@@ -398,6 +398,10 @@ impl VALUE {
         unsafe { rb_obj_frozen_p(self) != VALUE(0) }
     }
 
+    pub fn is_too_complex(self) -> bool {
```

Comment:
Nitpick: this method should probably be named `shape_too_complex()`

Reply:
Thanks, I've fixed it!